internal_worker_base.go

```diff
@@ -36,8 +36,8 @@ import (
 )
 
 const (
-	retryPollOperationInitialInterval    = time.Millisecond
-	retryPollOperationMaxInterval        = 1 * time.Second
+	retryPollOperationInitialInterval    = 20 * time.Millisecond
+	retryPollOperationMaxInterval        = 60 * time.Second
```

Comment:
This fix didn't help for the memory leak. but i am going still land this adjustments, will be helpful.

Changed the max to 10 seconds, we don't use the retryPollOperationExpirationInterval in poll so didn't change it. Since poll anyway is retrying we didn't use the policy retrier. We have log for each poll failure and that is too verbose to look at it.

Added metric to differentiate between transient and non-transient poll failures.